content/rs/security/certificates/ocsp-stapling.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Enable OCSP stapling
+linkTitle: Enable OCSP stapling
+description: Use OCSP stapling to verify certificates maintained by a third-party CA and authenticate connection attempts between clients and servers.
+weight: 50
+alwaysopen: false
+categories: ["RS"]
+aliases: ["/rs/security/ocsp-stapling"]
+---
+
+OCSP ([Online Certificate Status Protocol](https://en.wikipedia.org/wiki/Online_Certificate_Status_Protocol)) lets a client or server verify the status (valid, revoked, or unknown) of a certificate maintained by a third-party [certificate authority (CA)](https://en.wikipedia.org/wiki/Certificate_authority).
+
+When a client or server needs to check whether a certificate is still valid or has been revoked, it can send a request to the CA's OCSP server (also called an OCSP responder). The OCSP responder checks the certificate's status in the CA's [certificate revocation list](https://en.wikipedia.org/wiki/Certificate_revocation_list) and sends the status back as a signed and timestamped response.
+
+## OCSP stapling overview
+
+ With OCSP enabled, the Redis Enterprise server regularly polls the CA's OCSP responder for the certificate's status. After it receives the response, the server caches this status until its next polling attempt.
+
+ When a client tries to connect to the Redis Enterprise server, they perform a [TLS handshake](https://en.wikipedia.org/wiki/Transport_Layer_Security#TLS_handshake) to authenticate the server and create a secure, encrypted connection. During the TLS handshake, [OCSP stapling](https://en.wikipedia.org/wiki/OCSP_stapling) lets the Redis Enterprise server send (or "staple") the cached certificate status to the client.
+
+If the stapled OCSP response confirms the certificate is still valid, the TLS handshake succeeds and the client connects to the server.
+
+The TLS handshake fails and the client blocks the connection to the server if the stapled OCSP response indicates either:
+
+- The certificate has been revoked.
+
+- The certificate's status is unknown. This can happen if the OCSP responder fails to send a response.
+
+## Set up OCSP stapling
+
+Configure and enable OCSP stapling for your Redis Enterprise cluster with the [admin console](#admin-console-method), the [REST API](#rest-api-method), or [`rladmin`](#rladmin-method).
+
+After you enable OCSP, the server always staples the cached OCSP status. You also need to configure your client to use the stapled OCSP status.
```

Comment:
Does this mean you can't change your settings back after enabling OCSP?

Reply:
You can still turn off OCSP if you toggle the OCSP switch in the admin console or set ocsp_functionality to false with the REST API or rladmin. When ocsp_functionality equals true, the server always staples the OCSP status and sends it to the client, but it is the client's responsibility to actually use the OCSP staple. Maybe I need to reword this section a bit to clarify.

Reply:
Reworded this section for clarity.